src/js/App/Header/Tools.js

```diff
@@ -150,62 +148,59 @@ const Tools = () => {
   );
 
   return (
-    <PageHeaderTools widget-type="InsightsToolbar">
-      {/* Show tools on medium and above screens */}
-      <PageHeaderToolsGroup visibility={{ default: 'hidden', lg: 'visible' }}>
-        {isInternal && !window.insights.chrome.isProd && (
-          <PageHeaderToolsItem isSelected={window.insights.chrome.getBundle() === 'internal'}>{<InternalButton />}</PageHeaderToolsItem>
-        )}
-        {!isSettingsDisabled && <PageHeaderToolsItem>{<SettingsButton settingsMenuDropdownItems={settingsMenuDropdownItems} />}</PageHeaderToolsItem>}
-        <PageHeaderToolsItem>{<AboutButton />}</PageHeaderToolsItem>
-      </PageHeaderToolsGroup>
-
-      {/* Show full user dropdown on medium and above screens */}
-      <PageHeaderToolsGroup visibility={{ default: 'hidden', lg: 'visible' }}>
-        <PageHeaderToolsItem>
-          <UserToggle className="ins-c-dropdown__user" />
-        </PageHeaderToolsItem>
-      </PageHeaderToolsGroup>
+    <ToolbarGroup
+      className="pf-m-icon-button-group pf-m-align-right pf-m-spacer-none pf-m-spacer-md-on-md pf-u-mr-0"
+      alignment={{ default: 'alignRight' }}
+      spaceItems={{ default: 'spaceItemsNone' }}
+      widget-type="InsightsToolbar"
+    >
+      <ToolbarItem>{isBeta() ? <Badge className="chr-c-badge-beta">beta</Badge> : null}</ToolbarItem>
```

Comment:
no need for ternary operator: `isBeta() && <Badge ...`

Shouldn't we also hide the whole toolbar item? Otherwise it will be just empty